mimic/test/test_util.py

```diff
@@ -50,23 +55,28 @@ def test_seconds_to_timestamp_default_timestamp(self):
         default format string of ``%Y-%m-%dT%H:%M:%S.%fZ`` if no format
         string is provided.
         """
-        matches = [(0, "1970-01-01T00:00:00.000000Z"),
-                   (1.5, "1970-01-01T00:00:01.500000Z"),
-                   (121.4005, "1970-01-01T00:02:01.400500Z")]
-        for match in matches:
-            self.assertEqual(match[1], helper.seconds_to_timestamp(match[0]))
+        for seconds, timestamp in self.matches:
+            self.assertEqual(timestamp, helper.seconds_to_timestamp(seconds))
 
     def test_seconds_to_timestamp_provided_timestamp(self):
         """
         :func:`helper.seconds_to_timestamp` uses the provided timestamp format
         to format the seconds.
         """
-        matches = [("%m-%d-%Y %H:%M:%S", "01-01-1970 00:00:00"),
+        formats = [("%m-%d-%Y %H:%M:%S", "01-01-1970 00:00:00"),
                    ("%Y-%m-%d", "1970-01-01"),
                    ("%H %M %S (%f)", "00 00 00 (000000)")]
-        for match in matches:
-            self.assertEqual(match[1],
-                             helper.seconds_to_timestamp(0, match[0]))
+        for fmt, timestamp in formats:
+            self.assertEqual(timestamp,
+                             helper.seconds_to_timestamp(0, fmt))
+
+    def test_timestamp_to_seconds(self):
```

Comment:
Non-locking - Nova also accepts incomplete timestamps (e.g. `"1970-01-01"`, `"1970-01-01T00"`) - should we add tests to support those too?